box-setup/install-agent-clis.sh

```
@@ -0,0 +1,270 @@
+      "models": [
+        { "id": "glm-5.3-flash", "name": "GLM 5.3 Flash (API)", "contextWindow": 204800, "maxTokens": 16384, "reasoning": true, "input": ["text"] },
+        { "id": "glm-5.3", "name": "GLM 5.3 (API)", "contextWindow": 204800, "maxTokens": 16384, "reasoning": true, "input": ["text"] },
+        { "id": "glm-5.2", "name": "GLM 5.2 (API)", "contextWindow": 204800, "maxTokens": 16384, "reasoning": true, "input": ["text"] }
+      ]
+    }
+  }
+}
+JSON
+  note "wrote ~/.pi/agent/models.json"
+else
+  note "~/.pi/agent/models.json already present, left alone"
```

Shellcheck (0.11.0)

[warning] Tilde does not expand in quotes. Use $HOME.

(SC2088)